webrtc.html

```diff
@@ -4768,7 +4768,11 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
             nullable</dt>
             <dd>
               <p>The <code>track</code> attribute is the track that is
-              associated with this <code><a>RTCRtpSender</a></code> object.</p>
+              associated with this <code><a>RTCRtpSender</a></code> object.
+              If <code>track</code> is ended, or if
+              <code>track</code>.<var>muted</var> is set to <code>true</code>,
+              the <code>RTCRtpSender</code> sends silence (audio) or a black
```

Comment:
I would accept sending only 1 black video frame, but what about stop sending audio directly instead of wasting bandwidth sending silence?